build-tools/scripts/dependencies/linux-prepare-Ubuntu.sh

```diff
@@ -0,0 +1,26 @@
+UBUNTU_DEPS="autoconf \
+	autotools-dev \
+	automake \
+	clang \
+	curl \
+	g++-mingw-w64 \
+	gcc-mingw-w64 \
+	git \
+	libtool \
+	libz-mingw-w64-dev \
+	libzip4 \
+	linux-libc-dev \
+	make \
+	openjdk-8-jdk \
+	unzip \
+	vim-common"
+
+if [ $OS_ARCH = "x86_64" ]; then
+UBUNTU_DEPS="$UBUNTU_DEPS 
+	lib32stdc++6 \
+	lib32z1 \
+	libx32tinfo-dev \
+	linux-libc-dev:i386 \
+	zlib1g-dev:i386 "
```

Comment:
To avoid "unnecessary" changes, the final `"` should be alone on a newline

```bash
UBUNTU_DEPS="
    $UBUNTU_DEPS
    lib32stdc++6
    ...
    "
```